test/integration/features/proxy.feature

```diff
@@ -0,0 +1,87 @@
+@proxy @minishift-only
+Feature: Minishift can run behind proxy
+  As a user I can use Minishift behind a proxy.
+
```

Comment:
For now it is not using any environmental variables, but I am starting to think that it could probably use PROXY_PORT_NUMBER and maybe PROXY_IP, now detection of IP is fine on Fedora and Windows - but @gbraad is maybe having troubles with that, so adding IP and port as parameters is an option.

To your problem, I can imagine that if you have MINISHIFT_USER set, maybe minishift will ask you for password even if the box does not need the registration - tested interactive registration with nightly build of CDK so I am not sure with behaviour on minishift. Can try if I can replicate this idea. For this test I am using minishift 1.2.0.